data/2022/readme.md

```diff
@@ -52,3 +52,4 @@ Archive of datasets and articles from the 2022 series of `#TidyTuesday` events.
 | 46 | `2022-11-15` | [Web page metrics](2022-11-15) |  [httpArchive.org](https://httparchive.org/reports/loading-speed?start=earliest&end=latest&view=list) | [DataWrapper](https://blog.datawrapper.de/why-web-pages-can-have-a-size-problem/) & [Data is Plural](https://www.data-is-plural.com/archive/2022-11-02-edition/) |
 | 47 | `2022-11-22` | [UK Museums](2022-11-22) |  [MuseWeb by way of Data Is Plural](https://www.data-is-plural.com/archive/2022-10-19-edition/) | [MuseWeb Key Findings](https://museweb.dcs.bbk.ac.uk/findings) |
 | 48 | `2022-11-29` | [FIFA World Cup](2022-11-29) |  [Kaggle FIFA World Cup](https://www.kaggle.com/datasets/evangower/fifa-world-cup/) | [Dataset Notebooks](https://www.kaggle.com/datasets/evangower/fifa-world-cup/code) |
+| 49 | `2022-12-06` | [Elevators](data/2022/2022-12-06) |  [Elevators data](https://github.com/EmilHvitfeldt/elevators/tree/main/data-raw) | [Elevators data package and examples]([https://www.kaggle.com/datasets/evangower/fifa-world-cup/code](https://emilhvitfeldt.github.io/elevators/)) |
```

Comment:
Why is `https://www.kaggle.com/datasets/evangower/fifa-world-cup/code` listed here? :)

Reply:
Doh, thanks for the catch! Copy/paste issue!

Reply:
thanks! updated